composer.json

```diff
@@ -11,23 +11,18 @@
     },
     "name": "bestit/commercetools-odm-bundle",
     "description": "Makes the commercetools database accessible with the doctrine commons api.",
-    "extra": {
-        "branch-alias": {
-            "dev-master": "1.0-dev"
-        }
-    },
     "type": "lib",
     "require": {
-        "php": "^7.0",
-        "bestit/commercetools-odm": "^1.0@dev",
-        "symfony/dependency-injection": "^3.1 || ^4.0",
-        "symfony/yaml": "^3.1 || ^4.0",
-        "symfony/http-kernel": "^3.1 || ^4.0"
+        "php": "^7.2.5",
+        "bestit/commercetools-odm": "^0.55.3",
```

Comment:
What happened here?

Reply:
Symfony 5 requires 7.2.5. That was the right time to update our requirements.

And the odm lib without a concrete version has often caused conflicts. Each library, which has a concrete odm version as dependency, conflicts with this bundle.

Reply:
Updated to odm 1.0